R/result.R

```diff
@@ -23,8 +23,11 @@ result <- function(x, message = NULL, correct = FALSE) {
 }
 
 
-# TODO-document
-# Should be what EVERY check_* returns
+#' \code{grader_graded} object
```

Comment:
Graded submission value

Reply:
made it the first line of the docstring

Reply:
Remove line 27.  It it still being put into the title.